### ISSUE TITLE
Sidebar headers do not get  themed

### ISSUE THREAD

**Issue by pika-colony** [Nokse22/high-tide#8]
**Expected behavior:**
The entire application is themed when using themes, as is the case with all other GTK3/GTK4/Libadwaita apps I have tried.

**Actual behavior:**
Sections of application--noticeable especially on the sidebar--do not get themed, and instead retain their original Libadwaita dark color. 

I am using Gradience for theming, and do not have any issues with themes within other apps regardless. My first thought was potentially some deviation from the GTK4 spec, but I have provided screenshots below to see for yourself.

![image](https://github.com/Nokse22/high-tide/assets/169167220/f9b934c6-35ac-4826-a2b2-9f9b36ece906)
![image](https://github.com/Nokse22/high-tide/assets/169167220/f1fb5772-4444-443e-92c3-0a7edb832704)

**Comment by Nokse22** [comment]
Have you tried with Gradience nightly, because the current stable release doesn't theme the sidebar.

**Comment by pika-colony** [comment]
Yes, this is using the Gradience-git package from the AUR, which should be up to date with the repo itself.

**Comment by Nokse22** [comment]
So other libadwaita apps with a sidebar are themed correctly?

**Comment by pika-colony** [comment]
Yes, that is correct. Libadwaita and GTK4 apps seem to be themed correctly in my experience, including applications like Nautilus. I will spin up a virtual machine this weekend and see if I can reproduce the issue with a new configuration as well.

**Comment by pika-colony** [comment]
My apologies! I got a free moment to look through the issue reports for Gradience, and I came across [a similar issue](https://github.com/GradienceTeam/Gradience/issues/899). While their case was more severe, with the entire sidebar not being themed, [one of the comments](https://github.com/GradienceTeam/Gradience/issues/899#issuecomment-2169153339) provided a solution that fixed my issue with High Tide as well. Like I said, the lack of sidebar theming wasn't consistent across the rest of my Libadwaita apps, but for now it feels like the issue is Gradience's. Thanks for your time!
